packages/dialtone-icons/src/keywords-icons.json

```diff
@@ -489,7 +489,23 @@
       ],
       "whisper": [
         "soft"
-      ]
+      ],
+      "power-dialer": [
+        "powerdialer",
+        "dialer",
+        "lightning",
+        "list",
+        "speed"
+      ],
+      "power-dialer-off": [
+        "powerdialer",
+        "dialer",
+        "lightning",
+        "list",
+        "speed",
+        "off",
```

Comment:
Redundant.

Reply:
@fede-dp @francisrupert removing "off" is fine